Attribute not supported in HTML5 when run javadoc with JDK11

Hello, when I run "./gradlew javadoc --stacktrace" using JDK 11.0.2-open or 11.0.18-zulu I get:

```
[user@localhost ReactiveX-RxJava]$ ./gradlew javadoc --stacktrace

> Task :javadoc
/tmp/ReactiveX-RxJava/src/main/java/io/reactivex/rxjava3/observers/package-info.java:23: error: attribute not supported in HTML5: summary
 * <table border="1" style="border-collapse: collapse;" summary="The available observer types.">
                                                        ^
/tmp/ReactiveX-RxJava/src/main/java/io/reactivex/rxjava3/subjects/package-info.java:21: error: attribute not supported in HTML5: summary
 * <table border="1" style="border-collapse: collapse;" summary="The available subject classes with their respective base classes and consumer interfaces.">
                                                        ^
2 errors
```

Based on w3c [doc](https://www.w3.org/TR/WCAG20-TECHS/H73.html)
`Note: In HTML5 the summary attribute is obsolete. Assistive technologies may or may not continue to support the attribute. Authors should consider alternatives and only use with caution.`

I created a [PR](https://github.com/ReactiveX/RxJava/pull/7534) solving this situation.

--- a/src/main/java/io/reactivex/rxjava3/observers/package-info.java
+++ b/src/main/java/io/reactivex/rxjava3/observers/package-info.java
@@ -20,7 +20,8 @@
  * <p>
  * Available observer variants
  * <br>
- * <table border="1" style="border-collapse: collapse;" summary="The available observer types.">
+ * <table border="1" style="border-collapse: collapse;">
+ * <caption>The available observer types.</caption>
  * <tr><td><b>Reactive type</b></td><td><b>Base interface</b></td><td><b>Simple</b></td><td><b>Disposable</b></td><td><b>Resource</b></td></tr>
  * <tr>
  *     <td>{@link io.reactivex.rxjava3.core.Observable Observable}</td>

--- a/src/main/java/io/reactivex/rxjava3/subjects/package-info.java
+++ b/src/main/java/io/reactivex/rxjava3/subjects/package-info.java
@@ -18,7 +18,8 @@
  * <p>
  * Available subject classes with their respective base classes and consumer interfaces:
  * <br>
- * <table border="1" style="border-collapse: collapse;" summary="The available subject classes with their respective base classes and consumer interfaces.">
+ * <table border="1" style="border-collapse: collapse;">
+ * <caption>The available subject classes with their respective base classes and consumer interfaces.</caption>
  * <tr><td><b>Subject type</b></td><td><b>Base class</b></td><td><b>Consumer interface</b></td></tr>
  * <tr>
  *     <td>{@link io.reactivex.rxjava3.subjects.Subject Subject}

